src/app/api/submissions/route.js

```
@@ -26,212 +24,128 @@ export async function POST(request) {
 
       if (actual !== expected) {
         verdict = "Wrong Answer";
-        executionTime = Date.now() - testStartTime;
+        break;
+      }
+    }
 
-        // Save wrong answer submission
-        // Save Wrong Answer to database
+    // If user is authenticated, attempt to save the submission (do not fail request on DB errors)
+    const authHeader = request.headers.get("authorization");
```

Biome (2.4.4)

[error] Shouldn't redeclare 'authHeader'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)